### ISSUE TITLE
Field <Select /> does not send initialData if it does not change the value

### ISSUE THREAD

**Issue by fitinge** [Rocketseat/unform#11]
I'm using the initialData for a `<Select />` field and it already appears with the selected value. When I change the select, the form is sent with the correct values, but if I leave default, the value of the field gets undefined

```
const selectExample = [
  {
    id: "id1",
    title: "Teste"
  },
  {
    id: "id2",
    title: "Teste2"
  }
];

const initialExample = {
  select: "id2"
};

return (
  <Form initialData={initialExample} onSubmit={this.handleSubmit}>
    <Select name="select" options={selectExample} />
    <br />
    <br />
    <button type="submit">Enviar</button>
  </Form>
);
```

<img width="344" alt="Captura de Tela 2019-05-13 às 19 36 09" src="https://user-images.githubusercontent.com/1730080/57658774-74778b80-75b6-11e9-91f1-743603f20b22.png">

<img width="312" alt="Captura de Tela 2019-05-13 às 19 36 22" src="https://user-images.githubusercontent.com/1730080/57658784-7a6d6c80-75b6-11e9-9434-e7765edd6abd.png">

**Comment by jpdemagalhaes** [comment]
After a quick look it seems that `defaultValue` on `<select>` doesn't trigger `onChange` therefore, remaining undefined until a user input.

I'll take a look into into.

**Comment by jpdemagalhaes** [comment]
This was solved in #14. I'll be closing this.

Thanks 😄
